tests/ui/asm/inline-syntax.rs

```diff
@@ -1,11 +1,11 @@
-// revisions: x86_64 arm
-//[x86_64] compile-flags: --target x86_64-unknown-linux-gnu
-//[x86_64] check-pass
+//@revisions: x86_64 arm
+//@[x86_64]compile-flags: --target x86_64-unknown-linux-gnu
+//@[x86_64]check-pass
```

Comment:
I think the one space after `]` is allowed to stay, but it doesn't matter much in either direction

Reply:
Oh yep, any amount of whitespace (such that it would be removed by `trim_start`) is allowed. Since we're converting the entire test suite, this is a chance to effectively set the style that people should be using, since people are going to copy paste.  Should we do `//@[rev] command` with a single space then?

Reply:
Additionally we could normalize to `//@[rev] edition: 2021` such that there's a space after the `:` in a name-value directive?

Reply:
I like both of these